oshdb-util/src/test/java/org/heigit/bigspatialdata/oshdb/util/celliterator/IterateByTimestampsRelationsTest.java

```diff
@@ -92,6 +92,7 @@ public void testVisibleChange() {
     assertEquals(303, result.get(0).osmEntity.getChangesetId());
   }
 
+  @SuppressWarnings("unused")
```

Comment:
maybe it's better to use `ResultOfMethodCallIgnored` suppression here and remove the unused local variable?

Reply:
`ResultOfMethodCallIgnored` is incorrect. The Problem is, that the assigned variable `result` is never used.

Reply:
removed the assignment.